tests/ui/collapsible_span_lint_calls.rs

```diff
@@ -0,0 +1,43 @@
+ #![deny(clippy::internal)]
+ #![feature(rustc_private)]
+
+ extern crate rustc_session;
+ extern crate rustc_errors;
+ extern crate rustc_lint;
+ extern crate syntax;
+ extern crate clippy_lints;
+
+use rustc_session::{declare_tool_lint, declare_lint_pass};
+use rustc_errors::Applicability;
+use rustc_lint::{EarlyContext, EarlyLintPass};
+use syntax::ast::Expr;
+use clippy_lints::utils::span_lint_and_then;
```

Comment:
How to import span_lint_and_then. This failed because `clippy_lints::utils` is private mod.

Reply:
I guess, this isn't possible. Can you just redefine this function here?